cluster/gce/configure-vm.sh

```diff
@@ -529,6 +529,12 @@ function salt-set-apiserver() {
 EOF
 }
 
+function salt-set-test-node() {
+  cat <<EOF >>/etc/salt/minion.d/grains.conf
+  test: true
```

Comment:
I don't know too much about salt but this feels risky because we're appending `test: true` in the context of surrounding indentation right? Don't know if there's a better way